### DISCUSSION TITLE
Error message: curlCode: 6, Couldn't resolve host name

### DISCUSSION THREAD

**Discussion by calvin2020** [apache/doris#23500] [A - General / Q&A]
Please help me, I'm testing Cold Hot Separation,the detail as below:
1)CREATE RESOURCE "log_s3"
CREATE RESOURCE "log_s3"
PROPERTIES
(
    "type" = "s3",
    "AWS_ENDPOINT" =  "http://192.168.137.28:9000",   ---minio API address
    "AWS_REGION" = "xxx",
    "s3.bucket" = "doris-repo",
    "s3.root.path" = "/doris-repo/cold",
    "AWS_ACCESS_KEY" =  "xxx",
    "AWS_SECRET_KEY"="xxx" ,
    "use_path_style" = "true"	
);
2)CREATE STORAGE POLICY log_policy_1day
CREATE STORAGE POLICY log_policy_1day
PROPERTIES(
    "storage_resource" = "log_s3",
    "cooldown_ttl" = "300"
);
3)add storage_policy for test_tt
alter table test_tt set ("storage_policy" = "log_policy_1day");
the I check my doris be log,It shows a lot of error:
Error message: curlCode: 6, Couldn't resolve host name
0 response headers:
W0825 16:56:34.678841 396963 s3_util.cpp:69] [AWSClient] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
W0825 16:56:34.678850 396963 s3_util.cpp:69] [AWSClient] Request failed, now waiting 6400 ms before attempting again.
E0825 16:56:34.679873 397072 s3_util.cpp:66] [CurlHttpClient] Curl returned error code 6 - Couldn't resolve host name
E0825 16:56:34.679925 397072 s3_util.cpp:66] [AWSClient] HTTP response code: -1
Resolved remote host IP address: 
Request ID: 
Exception name: 
Error message: curlCode: 6, Couldn't resolve host name
0 response headers:
W0825 16:56:34.679937 397072 s3_util.cpp:69] [AWSClient] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
W0825 16:56:34.679944 397072 s3_util.cpp:69] [AWSClient] Request failed, now waiting 6400 ms before attempting again.
E0825 16:56:34.680109 396967 s3_util.cpp:66] [CurlHttpClient] Curl returned error code 6 - Couldn't resolve host name
E0825 16:56:34.680146 396967 s3_util.cpp:66] [AWSClient] HTTP response code: -1
Resolved remote host IP address: 
Request ID: 
Exception name: 
Error message: curlCode: 6, Couldn't resolve host name
0 response headers:
W0825 16:56:34.680156 396967 s3_util.cpp:69] [AWSClient] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
W0825 16:56:34.680162 396967 s3_util.cpp:69] [AWSClient] Request failed, now waiting 6400 ms before attempting again.
E0825 16:56:34.681345 396962 s3_util.cpp:66] [CurlHttpClient] Curl returned error code 6 - Couldn't resolve host name
E0825 16:56:34.681380 396962 s3_util.cpp:66] [AWSClient] HTTP response code: -1

Can you help me to check this issue? I have no idea
My doris version is 2.0.0

thanks

**Comment by catpineapple** [comment]
This problem will be solved in the next version. the PR is [https://github.com/apache/doris/pull/22938](https://github.com/apache/doris/pull/22938)
